src/App.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Routes, Route, Link } from 'react-router-dom';
+import './App.css';
+import CategoryPage from './components/CategoryPage';
+import BookPage from './components/BookPage';
+
+const App = () => (
+  <div className="App">
+    <nav>
+      <h1> Bookstore CMS</h1>
+      <ul>
+        <li>
+          <Link
+            to="/"
+            className="active"
+          >
+            Books
+          </Link>
+        </li>
+        <li>
+          <Link
+            to="/categories"
+          >
+            CATEGORIES
+          </Link>
+        </li>
+      </ul>
+    </nav>
```

Comment:
[OPTIONAL] You can write cleaner code and better structuring if you moved all this code to a `Navbar.js` component under `./components`, then simply import it here and render the component as `<Navbar />`.